app/views/users/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1><%= @user.name %></h1>
+<p><%= @user.bio %></p>
+<%= image_tag @user.photo %>
+
+<h2>Recent Posts</h2>
+<% @recent_posts.each do |post| %>
+  <div>
+    <h3><%= link_to post.title, user_post_path(@user, post) %></h3>
+    <p>Likes: <%= post.likes_counter %></p>
+    <p>Comments: <%= post.comments_counter %></p>
+  </div>
+<% end %>
```

Comment:
- Good job done so far, but according to the project requirements `View files should include only placeholders written in plain HTML, e.g: Here is a list of posts for a given user`. Please fix this for all view files 🙏